.github/workflows/build.yaml

```diff
@@ -337,28 +343,49 @@ jobs:
         id: release_drafter
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Upload Binary Linux
-        id: upload-binary-linux
+      - name: Upload Binary Linux Amd64
+        id: upload-binary-linux-amd64
         uses: gfreezy/upload-release-asset@v1.0.2
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           release_id: ${{ steps.release_drafter.outputs.id }}
           upload_url: ${{ steps.release_drafter.outputs.upload_url }}
-          asset_path: linux/kube-linter-linux
-          asset_name: kube-linter-linux
+          asset_path: linux/amd64/kube-linter-linux-amd64
+          asset_name: kube-linter-linux-amd64
```

Comment:
this will be a breaking change all users who download with a script need to update

Reply:
Do you want to upload a `kube-linter-linux` as well as a `kube-linter-linux-amd64` for backwards compatibility?

Reply:
I think we should keep it backward compatible for a one or two releases